docs/getting-started/accessing-resources/getting-access-to-data.md

```diff
@@ -120,3 +130,95 @@ This will download test data and direct `data/current` symlink to the test data
 
 To switch back to using ScPCA data, rerun the script with the `--release current` option.
 If you already have the most recent data, this will not repeat downloading the data you already have.
+
+
+## Accessing ScPCA module results
+
+Once an analysis modules matures, the Data Lab ports the analysis to a separate repository called [`OpenScPCA-nf`](https://github.com/AlexsLemonade/OpenScPCA-nf).
+This repository holds a [Nextflow workflow](https://www.nextflow.io/) to reproducibility run and generate results from completed analysis modules.
+
+If you wish to use output from a completed analysis module in your module, you will need to obtain the results generated by this workflow.
```

Comment:
I do think we want to say something about using results from other modules that are _not_ in the workflow yet. I'm not sure we have a specific rec here, but it is something we should address. Essential, I expect an FAQ of "What if the results I want to use are from a module that isn't in the workflow yet?"

Reply:
fyi https://github.com/AlexsLemonade/OpenScPCA-analysis/issues/515